### PR TITLE
[Mobile] Modified More block to be read-only

### PR DESCRIPTION
**Pull request by etoledom**
gutenberg-mobile side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1079

As mentioned in https://github.com/wordpress-mobile/gutenberg-mobile/issues/859#issuecomment-498321249, I opted for modifying the `More` block to be read only, in order for it to be ready for the next release.

I did a fast test using RichText. It does work well on iOS, but it had some alignment issues on Android, that would need native code to be fixed. It also had a visual glitch on split.

Fixing those issues on Android, it looks promising to use RichText to make this component editable.
Alternatively, we might also want to think on a simpler PlaneText component that we can manipulate at the native level for these cases.

On the design side, I noticed on web that More block and Page Break block look the same, so I implemented the same design we use for Page Break on mobile too.

more-block

To test:
- Run the app.
- Search for the Read More block in the example content (or add one if it's a new post)
- Check that it's not editable.
- Check that it can be selectable by tapping anywhere in the block.
- Paste this in the HTML mode:
```
<!-- wp:more {"customText":"Read more custom from web"} -->
<!--more Read more custom from web-->
<!-- /wp:more -->
```
- Go back to Visual mode.
- Check that it shows custom content.

### CODE REVIEW

File: packages/block-library/src/more/edit.native.js [hunk 1]
```diff
@@ -2,85 +2,47 @@
...
-					multiline={ true }
-					underlineColorAndroid="transparent"
-					onChange={ this.onChangeInput }
-					placeholder={ defaultText }
-					isSelected={ this.props.isSelected }
-					onFocus={ onFocus }
-					onBlur={ onBlur }
+			<View
+				accessible={ ! this.props.isSelected }
+				accessibilityLabel={ __( 'More block' ) }
```
Comment (koke) [inline packages/block-library/src/more/edit.native.js]: Should this be "Read more block"? More feels a bit ambiguous

**Review by koke — APPROVED** [review]
Looking 😍 to me

File: packages/block-library/src/more/edit.native.js (hunk 1 above)
Comment (etoledom) [reply packages/block-library/src/more/edit.native.js]: Modified the accessibility config based on https://github.com/wordpress-mobile/gutenberg-mobile/pull/937 

Now all blocks will have a default label based on the gutenberg-defined block name.

**Comment by etoledom** [comment]
Thank you!

**Pull request merged**